docs/ossm/kiali/README.md

```diff
@@ -169,7 +185,7 @@ The following steps show how to install Kiali via the OpenShift CLI.
     [!NOTE]
     The `openshift` authentication strategy is the only supported authentication configuration when Kiali is deployed with OpenShift Service Mesh (OSSM). The `openshift` strategy controls access based on the individual’s role-based access control (RBAC) roles of the OpenShift.
 
-    By default, the Kiali Operator will install the Kiali Server whose version is the same as the operator itself. You can ask the operator to install an earlier version of the Kiali Server by specifying the `spec.version` field to indicate which version of the Kiali Server to install (check the documentation for the valid versions that are supported by the operator and which Istio versions work with which Kiali versions):
+    By default, the Kiali Operator will install the Kiali Server whose version is the same as the operator itself. You can ask the operator to install an earlier version of the Kiali Server by specifying the `spec.version` field to indicate which version of the Kiali Server to install (check [here](#ossm-version-compatibility) for the valid versions that are supported by the operator and which OSSM versions work with which Kiali Server versions). The version to install has to be in the form `vX.Y`:
```

Comment:
See my earlier comment about the "v" prefix and the "no patch number" in the spec.version field.